### DISCUSSION TITLE
ImageGlass 9.0.7.1125 - Official Release

### DISCUSSION THREAD

**Discussion by d2phap** [d2phap/ImageGlass#1700] [General]
<img width="640" alt="ImageGlass 9.0.7.1125" src="https://raw.githubusercontent.com/ImageGlass/releases/main/screenshots/v9.0/9.0_a.webp">

[![Downloads](https://img.shields.io/github/downloads/d2phap/imageglass/9.0.7.1125/total?color=%232A7C91&label=downloads&style=for-the-badge)](https://imageglass.org/release/imageglass-9-0-7-1125-46)

Please support this project by making a donation or purchasing the Microsoft Store version (version 8):

<a href="https://www.microsoft.com/store/productId/9N33VZK3C7TH?cid=GitHubRelease&referrer=appbadge&source=GithubD2phap">
  <img height="69" src="https://user-images.githubusercontent.com/3154213/197671083-4099d80f-07fc-4af9-b80d-6357f0968347.svg" />
</a>

## 🪁 System Requirements
- Windows 10/11 64-bit, version 2004 or later
- [.NET Desktop Runtime 8 (64-bit)](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
- [WebView2 Runtime 64-bit](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section)

## 📝 Changelog (since [ImageGlass 8.10](https://github.com/d2phap/ImageGlass/releases/tag/8.10.9.27))
Please read the blogpost: https://imageglass.org/news/introducing-the-new-imageglass-version-9-88

### 🌱 New:
- Updated app icon and the built-in theme packs
- Added new setting `LastOpenedSetting` to save the last opened setting tab
- Added "Export language pack" in app settings > Language tab
- Added available image information tags in app settings > General tab

### 🔮 Improvements:
- Upgraded Magick.NET to v13.4.0
- Corrected the `ImageInterpolation` setting value from `SampleLinear` to `MultiSampleLinear`. If you encounter error "Could not load user settings" after upgrading, you can edit the new value in `igconfig.json` file.
- Added <kbd>Ctrl+W</kbd> as an additional hotkey for closing the app (#1667)
- Now uses the toolbar background color for the window title bar when the backdrop style is set to `None`
- Will not reload the image if the folder contains only 1 image when users view the next/previous image
- Improved the opening time for Print Pictures dialog
- Improved the time taken to save image files
- Improved the opening time for image files larger than 16380 pixels by implementing `PhotoSauce.MagicScaler` to scale the image file
- Updated the image information on the title bar:
  + Now displays the rendered and actual image dimensions if the image file is larger than 16380 pixels
  + Used a vertical em dash (U+FE31) as the image info separator on the title bar
- App settings:
  + Moved the Slideshow background setting from the Appearance tab to the Slideshow tab
  + Added a tooltip for each textbox in the app settings to help the user input a valid value
  + Moved the position of the "Get help" button from the top right to the bottom left corner
- Updated the Quick Setup dialog to follow the user's language and theme settings

### 🐛 Bugfixes:
- Fixed an issue where ImageGlass didn't save the Color profile setting (#1669)
- Fixed an issue where ImageGlass failed to remember its window's position when anchored (#1646)
- Fixed an issue where app settings window is not functional if the installed path contained an apostrophe (`'`) (#1685)
- Fixed an issue where ImageGlass did not render correctly when using a theme pack with a transparency color
- Fixed an issue where opening a photo in single-instance mode caused random clicks on other apps
- Fixed an issue where pressing <kbd>Ctrl+C</kbd> multiple times resulted in an unhandled exception
- Fixed an issue where the user couldn't select an image area to crop when navigation buttons were visible
- Fixed an issue where layout buttons in the app settings were not updated when changing the select box or text box
- Fixed an issue where the language of the Aspect ratio in the Crop tool and the Image information on the title bar was not updated
- Fixed an issue where the zoom value is reset when viewing a single image frame
- Fixed an issue where Window Fit mode does not scale the window correctly when enabling Page Navigation tool is visible
- Fixed an issue where the buttons in the toolbar overflow dropdown does not show tooltip on hovering
- Fixed an issue where users cannot install language pack if "Language" folder does not exist (#1688)
- Fixed an issue where users cannot use "Kobe-Light" theme if the startup directory is not writable
- Fixed an issue where saving clipboard image crashes the app

## 🟣 Other updates for ImageGlass 9.0
- Published [ImageGlass.Tools library v1.9070](https://github.com/ImageGlass/ImageGlass.Tools)
- Published [ExifGlass app v1.3](https://github.com/d2phap/ExifGlass)

**Full Changelog**: https://github.com/d2phap/ImageGlass/compare/9.0.6.1008...9.0.7.1125

<hr /><em>This discussion was created from the release <a href='https://github.com/d2phap/ImageGlass/releases/tag/9.0.7.1125'>ImageGlass 9.0.7.1125 - Official Release</a>.</em>

**Comment by ghost** [comment]
So version 9 now requires MS Edge WebView??? I purged anything related to Edge and WebView from my system so I guess this means back to IrfanView...

**Reply by d2phap** [reply]
It's been announced for a half year. Webview2 is the UI for the Settings, Quick Setup, About, Check for Update dialog. You can use ImageGlass 9 without Webview2 (need to bypass the Quick Setup dialog), or feel free to use the version 8 or other apps.

**Comment by ghost** [comment]
after rotating image, Save button automatically appear right side of the folder icon & even save button enable in toolbar (2 save button)

main problem is: toolbar shifted to left side,
.           so problem in click, click, click, click again & again in rotate button

![Untitled-12](https://github.com/d2phap/ImageGlass/assets/120308983/aeeeadff-b100-48a2-b2b5-1fae2fb52012)

**Reply by d2phap** [reply]
It's by design. Rotating or Flipping does not auto-save the image until you manually save it. The Save button on the right is to remind user to save the image.

**Reply by ghost** [reply]
poblem is not for this button, problem is tollbar shifted to left side..

so if i click again & again rotate butten, Its not possible with same mouse position.. (need to change mouse position left-right)

**Reply by ghost** [reply]
**solution:** toolbar should be not shift to left side, if have some amount of empty space in toolbar

**Comment by ghost** [comment]
Overall this update is Very Good, but my [HR Image problem](https://github.com/d2phap/ImageGlass/issues/1687) still exist...

**Reply by d2phap** [reply]
Sorry, I didn't have time to investigate it.

**Reply by ghost** [reply]
when you get some time, then investigate.

**Comment by siblount** [comment]
YESSSSS!!! Hardware acceleration baby!!! Best update. Good work!

**Comment by ghost** [comment]
@d2phap as you know about my PC (intigrated Graphics i5 1stGen)..

Getting Imageglass Speed to open
Version 8: 2-3 seconds
Version 9: 3-5 seconds

**Reply by d2phap** [reply]
For low spec machine, I suggest using v8 or other apps.
